frontend/views/containers/proposals/ProposalsWidget.vue

```diff
@@ -150,6 +126,15 @@ export default ({
     }
   },
   methods: {
+    onProposalArchived (hashWithProp) {
+      this.ephemeral.archivedProposals.unshift(hashWithProp)
+      // after a day, remove it from the list
+      setTimeout(() => {
+        this.ephemeral.archivedProposals = this.ephemeral.archivedProposals.filter(x => {
+          return x[0] !== hashWithProp[0]
+        })
+      }, 1000 * 60 * 60 * 24) // 1 day
```

Comment:
Shouldn't these timeouts be cleared before destroying or unmounting the component? I'm afraid too many timers will remain active for example after logging out and in again several times

Reply:
Good catch!